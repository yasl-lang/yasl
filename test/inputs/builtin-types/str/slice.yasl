##[2, 3]\n23\n[1, 2, 3, 4]\n1234\n[1, 2, 3, 4, 5]\n12345\n

const y = "12345"

echo y[1:3]
echo y[0:-1]
echo y[-10:20]

let letters = "abcdef"

echo letters[:3]
echo letters[3:]
echo letters[:]
