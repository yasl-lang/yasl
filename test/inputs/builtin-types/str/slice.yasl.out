23
1234
12345
abc
def
abcdef
